kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -51,15 +51,14 @@
       </KGridItem>
 
       <KGridItem
-        style="position: relative; right: 0; padding: 0 0.5em 0 1em; text-align: right"
-        :layout4="{ span: 2 }"
+        :layout4="{ span: 4 }"
         :layout8="{ span: 3 }"
-        :layout12="{ span: 2 }"
+        :layout12="{ span: 4 }"
+        class="add-more-button"
```

Comment:
Maybe `add-more-button-container` as this class is for the KGridItem not the contained KButton?